compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```
@@ -4371,6 +4371,22 @@
                     "type": "ExpressionNode"
                 }
             ]
+        },
+        {
+            "name": "TupleMemberDescriptorNode",
+            "base": "TypeDescriptorNode",
+            "kind": "TYPE_DESC",
+            "attributes": [
+                {
+                    "name": "annotation",
+                    "type": "AnnotationNode",
+                    "occurrences": "MULTIPLE"
+                },
+                {
+                    "name": "typeDesc",
+                    "type": "Node"
```

Here we can use the `TypeDescriptorNode` abstraction as whatever the node comes here is a `TypeDescriptorNode`.
```suggestion
                    "name": "typeDescriptor",
                    "type": "TypeDescriptorNode"
```